doc/subsystems/networking/gptp_release_notes.rst

```diff
@@ -0,0 +1,55 @@
+gPTP stack for Zephyr
+#####################
+
+Overview
+********
+
+This gPTP stack supports the protocol and procedures as defined in
+the IEEE 802.1AS-2011 standard (Timing and Syncronization for
+Time-Sensitive Applications in Bridged Local Area Networks).
+
+Supported features
+*******************
+
+The stack handles communications and state machines defined in the standard
+IEEE802.1AS-2011. Mandatory requirements, for a full-duplex point-to-point link
```

Comment:
Commas are a bit off:

    Mandatory requirements for a full-duplex point-to-point link
    endpoint, as defined in Annex A of the standard, are supported.